x-pack/plugins/data_visualizer/public/application/common/components/results_links/results_links.tsx

```diff
@@ -12,10 +12,23 @@ import { EuiFlexGroup, EuiFlexItem, EuiCard, EuiIcon } from '@elastic/eui';
 import { TimeRange, RefreshInterval } from '@kbn/data-plugin/public';
 import { FindFileStructureResponse } from '@kbn/file-upload-plugin/common';
 import type { FileUploadPluginStart } from '@kbn/file-upload-plugin/public';
+import { flatten } from 'lodash';
+import { LinkCardProps } from '../link_card/link_card';
 import { useDataVisualizerKibana } from '../../../kibana_context';
+import { isDefined } from '../../util/is_defined';
 
 type LinkType = 'file' | 'index';
 
+export interface AsyncLinkCardParams {
+  dataViewId: string;
+  dataViewTitle?: string;
+  globalState?: any;
+}
+
+export type AsyncLinkCards = Array<
```

Comment:
I'm not sure about the name `AsyncLinkCards` 
It implies that there are also non-async links cards.
The functions that return the `ResultsLinks` are all async, so adding the word `async` to the name seems unnecessary.
Maybe something like `GetResultLinks` would be more accurate? Although i'm not crazy about that name.

Reply:
Updated here https://github.com/elastic/kibana/pull/131342/commits/966ff9f91c5ec282e16201a0f76f2e248aed2a86